src/components/SearchAndFilter.js

```diff
@@ -126,27 +133,31 @@ class SearchAndFilter extends React.Component {
           centered="true"
           size="lg"
         >
-          <Modal.Body style={styles.formBody}>
+          <Modal.Body className="formBody">
             <Form>
-              <label>Name: </label>
-              <input type="text" value={this.state.name} onChange={this.setName} />
+              <label>Name: 
+                <input type="text" value={this.state.name} onChange={this.setName} />
+              </label>
               <br></br>
               <label>Ingredients: </label>
-              <Button onClick={this.addInput} type="button">+</Button>
+              <button id="form-add" onClick={this.addInput} type="button">+</button>
               <br></br>
               {this.state.inputFields}
               <br></br>
-              <label>Min Time: </label>
-              <input type="number" value={this.state.minTime} onChange={this.setMin} />
+              <label>Min Time:
+                <input type="number" value={this.state.minTime} onChange={this.setMin} />
+              </label>
               <br></br>
-              <label>Max Time: </label>
-              <input type="number" value={this.state.maxTime} onChange={this.setMax} />
+              <label>Max Time:
+                <input type="number" value={this.state.maxTime} onChange={this.setMax} />
+              </label>
               <br></br>
-              <label>Created By: </label>
-              <input type="text" value={this.state.createdBy} onChange={this.setCreator} />
+              <label>Created By:
```

Comment:
can be resolved in another pull request (file an issue): Do you want Created by to be filled in by the user? I'd assume this is auto-filled by the server from their login information...